### PR TITLE
Makes EVE API key access and usage more readable.

### PR DESCRIPTION
**Pull request by Acidity**
No longer requires the use of bitwise operators in areas other than the key subdirectory, adds functions to check whether a certain mask supports any of the various eve api functions, and more!

### CODE REVIEW

**Comment by Acidity** [comment]
Still want to double check that I hit all of the places that this affects before merging.

**Comment by jboning** [comment]
https://github.com/bravecollective/core/blob/develop/brave/core/character/model.py#L199
https://github.com/bravecollective/core/blob/develop/brave/core/util/signal.py#L54

File: brave/core/character/model.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def titlesort(i):
     def credential_for(self, mask):
         """Return the least-permissive API key that can satisfy the given mask."""
         
-        candidates = [i for i in self.credentials if not mask or not i.mask or i.mask & mask == mask]
+        candidates = [i for i in self.credentials if not mask or not i.mask or i.mask.has_access(mask)]
```
Comment (jboning) [inline brave/core/character/model.py:202]: Below, `bin(candidate.mask).count('1')` is still going to break

**Comment by jboning** [comment]
https://github.com/bravecollective/core/blob/develop/brave/core/group/acl.py#L191 one more I think. sorry for not catching it the first time around :(

File: brave/core/key/controller.py [hunk 2]
```diff
@@ -102,7 +102,6 @@ def post(self, **kw):
         
         try:
             record.save()
-            record.pull()
```
Comment (jboning) [inline brave/core/key/controller.py:105]: looking again, I think maybe we do want to have this here. The pull triggered by the save will be asynchronous, so we won't necessarily have the character list ready to return to the web interface.

**Pull request merged**